learning_wavelets/models/learnlet_layers.py

```diff
@@ -203,7 +223,7 @@ def get_config(self):
 
 class LearnletSynthesis(Layer):
     __name__ = 'learnlet_synthesis'
-    def __init__(self, normalize=True, n_scales=4, n_channels=1, synthesis_use_bias=False, synthesis_norm=False, res=False, kernel_size=5, wav_type='starlet'):
+    def __init__(self, normalize=True, n_scales=5, n_tiling=256, n_channels=1, synthesis_use_bias=False, synthesis_norm=False, res=True, kernel_size=13, wav_type='starlet', n_shearlets=85):
```

Comment:
Let's not change the defaults you don't need for backward compatibility's sake
```suggestion
    def __init__(self, normalize=True, n_scales=4, n_tiling=256, n_channels=1, synthesis_use_bias=False, synthesis_norm=False, res=False, kernel_size=5, wav_type='starlet', n_shearlets=85):
```